src/ui/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { RequestSpec } from '../RequestSpec';
+
+export const labelForRequestSpecKey = (key: keyof RequestSpec, rq: Partial<RequestSpec>): string => {
```

Comment:
can you introduce this in a commit before this commit?
`src/ui/QueryEditor.tsx` also defines those labels, I think we should unify this.
Maybe we can make `requestSpecKey` a required field for `CheckMkSelectNegatable`, `Filter`, `CheckMkSelect`, `CheckMkSelectNegatable` and add the label in each of those components with the help of this function?

Reply:
I opted against adding to the Select since that thing gets to complicated for my taste, but another commit that also modifies the QueryEditor is a good Idea.